### ISSUE TITLE
All Jackett Indexers warning despite it being disabled

### ISSUE THREAD

**Issue by guipace** [Sonarr/Sonarr#5848]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

I am getting the `Indexers using the unsupported Jackett 'all' endpoint: All Jackett Indexers` warning because I have that configured. However, it's disabled and not being used.

<img width="1580" alt="image" src="https://github.com/Sonarr/Sonarr/assets/63423828/1e0db338-7c98-44d0-b06d-6e43c5833e45">

<img width="683" alt="image" src="https://github.com/Sonarr/Sonarr/assets/63423828/974588e1-8e93-44f7-b463-6e16de4f7037">

### Expected Behavior

No warning should be triggered if the offending indexer configuration is disabled. I have the same setup in Radarr and it doesn't generate that warning.

### Steps To Reproduce

1. Add the Jackett 'all' endpoint indexer
2. Disable it
3. See the warning persist in the system -> status page

### Environment

```markdown
- OS: Unraid 6.12.2
- Sonarr: 3.0.10.1567
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: Brave 1.56.9
```

### What branch are you running?

Main

### Trace Logs?

2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'firstdayofweek' defaultValue:'1'
2023-07-26 19:20:11.4|Trace|EventAggregator|ConfigFileSavedEvent <~ ReconfigureLogging
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'calendarweekcolumnheader' defaultValue:'ddd M/D'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'timeformat' defaultValue:'h(:mm)a'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'showrelativedates' defaultValue:'True'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'enablecolorimpairedmode' defaultValue:'False'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxytype' defaultValue:'Http'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxyusername' defaultValue:''
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxypassword' defaultValue:''
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxybypasslocaladdresses' defaultValue:'True'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'backupfolder' defaultValue:'Backups'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'backupinterval' defaultValue:'7'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'backupretention' defaultValue:'28'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'certificatevalidation' defaultValue:'Enabled'
2023-07-26 19:20:11.4|Trace|EventAggregator|Publishing ConfigSavedEvent
2023-07-26 19:20:11.4|Trace|EventAggregator|ConfigSavedEvent ~> TaskManager
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxytype' defaultValue:'Http'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxyusername' defaultValue:''
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxypassword' defaultValue:''
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxybypasslocaladdresses' defaultValue:'True'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'certificatevalidation' defaultValue:'Enabled'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'backupfolder' defaultValue:'Backups'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'rsssyncinterval' defaultValue:'15'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'backupinterval' defaultValue:'7'
2023-07-26 19:20:11.4|Trace|HttpClient|Req: [GET] http://localhost:8585/api/v2/app/preferences
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'backupinterval' defaultValue:'7'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'backupretention' defaultValue:'28'
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxytype' defaultValue:'Http'
2023-07-26 19:20:11.4|Trace|Http|Res: 19428 [PUT] /api/v3/config/host: 202.Accepted (91 ms)
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxybypasslocaladdresses' defaultValue:'True'
2023-07-26 19:20:11.4|Trace|EventAggregator|ConfigSavedEvent <~ TaskManager
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxyusername' defaultValue:''
2023-07-26 19:20:11.4|Debug|Api|[PUT] /api/v3/config/host: 202.Accepted (91 ms)
2023-07-26 19:20:11.4|Trace|ConfigService|Using default config value for 'proxypassword' defaultValue:''
2023-07-26 19:20:11.4|Trace|HttpClient|Res: [GET] http://localhost:8585/api/v2/app/preferences: 200.OK (4900 bytes) (10 ms)
2023-07-26 19:20:11.4|Trace|HttpClient|Response content (4900 bytes): {"add_trackers":"","add_trackers_enabled":false,"alt_dl_limit":10240,"alt_up_limit":10240,"alternative_webui_enabled":true,"alternative_webui_path":"/data/theme/vuetorrent","announce_ip":"","announce_to_all_tiers":true,"announce_to_all_trackers":false,"anonymous_mode":false,"async_io_threads":10,"auto_delete_mode":0,"auto_tmm_enabled":true,"autorun_enabled":false,"autorun_on_torrent_added_enabled":false,"autorun_on_torrent_added_program":"","autorun_program":"","banned_IPs":"","bittorrent_protocol":0,"block_peers_on_privileged_ports":false,"bypass_auth_subnet_whitelist":"","bypass_auth_subnet_whitelist_enabled":false,"bypass_local_auth":true,"category_changed_tmm_enabled":false,"checking_memory_use":32,"connection_speed":30,"current_interface_address":"","current_network_interface":"","dht":true,"disk_cache":-1,"disk_cache_ttl":60,"disk_io_read_mode":1,"disk_io_type":0,"disk_io_write_mode":1,"disk_queue_size":1048576,"dl_limit":0,"dont_count_slow_torrents":false,"dyndns_domain":"changeme.dyndns.org","dyndns_enabled":false,"dyndns_password":"","dyndns_service":0,"dyndns_username":"","embedded_tracker_port":9000,"embedded_tracker_port_forwarding":false,"enable_coalesce_read_write":false,"enable_embedded_tracker":false,"enable_multi_connections_from_same_ip":false,"enable_piece_extent_affinity":false,"enable_upload_suggestions":false,"encryption":0,"excluded_file_names":"","excluded_file_names_enabled":false,"export_dir":"","export_dir_fin":"","file_pool_size":5000,"hashing_threads":1,"idn_support_enabled":false,"incomplete_files_ext":false,"ip_filter_enabled":false,"ip_filter_path":"","ip_filter_trackers":false,"limit_lan_peers":true,"limit_tcp_overhead":false,"limit_utp_rate":true,"listen_port":42691,"locale":"en","lsd":true,"mail_notification_auth_enabled":true,"mail_notification_email":"","mail_notification_enabled":false,"mail_notification_password":"","mail_notification_sender":"qBittorrent_notification@example.com","mail_notification_smtp":"smtp.changeme.com","mail_notification_ssl_enabled":false,"mail_notification_username":"","max_active_checking_torrents":1,"max_active_downloads":3,"max_active_torrents":5,"max_active_uploads":3,"max_concurrent_http_announces":50,"max_connec":500,"max_connec_per_torrent":100,"max_ratio":1,"max_ratio_act":0,"max_ratio_enabled":true,"max_seeding_time":-1,"max_seeding_time_enabled":false,"max_uploads":20,"max_uploads_per_torrent":4,"memory_working_set_limit":512,"outgoing_ports_max":0,"outgoing_ports_min":0,"peer_tos":4,"peer_turnover":4,"peer_turnover_cutoff":90,"peer_turnover_interval":300,"performance_warning":false,"pex":true,"preallocate_all":false,"proxy_auth_enabled":false,"proxy_hostname_lookup":true,"proxy_ip":"0.0.0.0","proxy_password":"(removed)","proxy_peer_connections":false,"proxy_port":8080,"proxy_torrents_only":false,"proxy_type":0,"proxy_username":"(removed)","queueing_enabled":false,"random_port":false,"reannounce_when_address_changed":false,"recheck_completed_torrents":false,"refresh_interval":1500,"request_queue_size":500,"resolve_peer_countries":true,"resume_data_storage_type":"Legacy","rss_auto_downloading_enabled":false,"rss_download_repack_proper_episodes":true,"rss_max_articles_per_feed":50,"rss_processing_enabled":false,"rss_refresh_interval":30,"rss_smart_episode_filters":"s(\\d+)e(\\d+)\n(\\d+)x(\\d+)\n(\\d{4}[.\\-]\\d{1,2}[.\\-]\\d{1,2})\n(\\d{1,2}[.\\-]\\d{1,2}[.\\-]\\d{4})","save_path":"/data/complete","save_path_changed_tmm_enabled":true,"save_resume_data_interval":60,"scan_dirs":{},"schedule_from_hour":8,"schedule_from_min":0,"schedule_to_hour":20,"schedule_to_min":0,"scheduler_days":0,"scheduler_enabled":false,"send_buffer_low_watermark":10,"send_buffer_watermark":500,"send_buffer_watermark_factor":50,"slow_torrent_dl_rate_threshold":2,"slow_torrent_inactive_timer":60,"slow_torrent_ul_rate_threshold":2,"socket_backlog_size":30,"ssrf_mitigation":true,"start_paused_enabled":false,"stop_tracker_timeout":5,"temp_path":"/data/complete/temp","temp_path_enabled":false,"torrent_changed_tmm_enabled":true,"torrent_content_layout":"Original","torrent_stop_condition":"None","up_limit":0,"upload_choking_algorithm":1,"upload_slots_behavior":0,"upnp":true,"upnp_lease_duration":0,"use_category_paths_in_manual_mode":false,"use_https":false,"utp_tcp_mixed_mode":0,"validate_https_tracker_certificate":true,"web_ui_address":"*","web_ui_ban_duration":3600,"web_ui_clickjacking_protection_enabled":true,"web_ui_csrf_protection_enabled":false,"web_ui_custom_http_headers":"","web_ui_domain_list":"*","web_ui_host_header_validation_enabled":true,"web_ui_https_cert_path":"","web_ui_https_key_path":"","web_ui_max_auth_fail_count":5,"web_ui_port":8585,"web_ui_reverse_proxies_list":"","web_ui_reverse_proxy_enabled":false,"web_ui_secure_cookie_enabled":true,"web_ui_session_timeout":3600,"web_ui_upnp":false,"web_ui_use_custom_http_headers_enabled":false,"web_ui_username":"(removed)"}
2023-07-26 19:20:11.4|Trace|HttpClient|Req: [GET] https://services.sonarr.tv/v1/ping
2023-07-26 19:20:11.5|Trace|ConfigService|Using default config value for 'proxytype' defaultValue:'Http'
2023-07-26 19:20:11.5|Trace|ConfigService|Using default config value for 'proxybypasslocaladdresses' defaultValue:'True'
2023-07-26 19:20:11.5|Trace|ConfigService|Using default config value for 'proxyusername' defaultValue:''
2023-07-26 19:20:11.5|Trace|ConfigService|Using default config value for 'proxypassword' defaultValue:''
2023-07-26 19:20:14.4|Trace|Http|Req: 19429 [GET] /api/v3/health (from 10.11.212.31 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36)
2023-07-26 19:20:14.4|Trace|Http|Req: 19430 [GET] /api/v3/diskspace (from 10.11.212.31 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36)
2023-07-26 19:20:14.4|Trace|Http|Res: 19429 [GET] /api/v3/health: 200.OK (6 ms)
2023-07-26 19:20:14.4|Debug|Api|[GET] /api/v3/health: 200.OK (6 ms)
2023-07-26 19:20:14.4|Trace|Http|Req: 19431 [GET] /api/v3/system/status (from 10.11.212.31 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36)
2023-07-26 19:20:14.4|Trace|Http|Res: 19431 [GET] /api/v3/system/status: 200.OK (1 ms)
2023-07-26 19:20:14.4|Debug|Api|[GET] /api/v3/system/status: 200.OK (1 ms)
2023-07-26 19:20:14.4|Trace|Http|Res: 19430 [GET] /api/v3/diskspace: 200.OK (32 ms)
2023-07-26 19:20:14.4|Debug|Api|[GET] /api/v3/diskspace: 200.OK (32 ms)
2023-07-26 19:20:15.5|Trace|Http|Req: 19432 [GET] /api/v3/log/file (from 10.11.212.31 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36)
2023-07-26 19:20:15.5|Trace|Http|Res: 19432 [GET] /api/v3/log/file: 200.OK (4 ms)
2023-07-26 19:20:15.5|Debug|Api|[GET] /api/v3/log/file: 200.OK (4 ms)
2023-07-26 19:20:18.2|Trace|Http|Req: 19433 [GET] /logfile/sonarr.trace.txt (from 10.11.212.31 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36)
2023-07-26 19:20:18.2|Trace|Http|Res: 19433 [GET] /logfile/sonarr.trace.txt: 200.OK (2 ms)

**Comment by guipace** [comment]
> What Base URL and API path are you using?

Not sure I understand your question. Sonarr is running on localhost.

Indeed, I'm running v4 for Radarr. Isn't Sonarr v3 the latest stable release?
